docs/xxinitialrampercentage.md

```diff
@@ -35,7 +35,8 @@ These Oracle HotSpot options can be used to specify the initial and maximum size
 
 : Where N is a value between 0 and 100, which can be of type "double". For example, 12.3456.
 
-<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** If you set a value for [`-Xms`](xms.md), these options are ignored.
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** If you set a value for [`-Xms`](xms.md), the `-XX:InitialRAMPercentage` option is ignored.
+If you set a value for [`-Xmx`](xmx.md), the `-XX:MaxRAMPercentage` option is ignored.
```

Comment:
-Xms and Xmx are both described in the topic `xms.md`; there is no 'xmx.md' topic! Please fix second link, which is currently broken.  Same comment applies to 'docs/xxusecontainersupport.md'.
Should a similar comment be added to the `xms.md` topic as warning of the potential effect of setting one or both of those options?

Reply:
updated, thanks